backend/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+services:
+
+  mysqldb:
+    image: mysql:8
+    container_name: mysqldb
```

Comment:
for better readability name this skynestdb or something that will represent that this is tightly connected to SkyNest project. In future you may work on several projects and in that case you can get easily confused. Same applies for other services defined here

Reply:
Something like
mysqldb -> skynest-db
cassndra -> skynest-token-db
cassandra-load-keyspace -> skynest-token-db-load-keyspace

Reply:
sure, why not